runtime/src/app/app.component.ts

```diff
@@ -6,7 +6,7 @@ import { AuthenticationService } from 'ngx-login-client';
 import { GlobalSettings } from './shared/globals';
 
 @Component({
-  selector: 'alm-app',
+  selector: 'f8-app',
```

Comment:
Shouldn't it be f8-planner-app

Reply:
It's **f8-app** in fabric8-ui and we have to follow the same because now tests are written for both integrated and standalone. Changing it here will require in fabric8-ui as well.